pkgs/development/libraries/omorfi/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, autoreconfHook
+, cg3
+, fetchFromGitHub
+, hfst
+, hfst-ospell
+, icu
+, libvoikko
+, makeWrapper
+, pkg-config
+, python3
+, python3Packages
```

Comment:
```suggestion
```

Reply:
@SuperSandro2000 just out of curiosity, is there any reason to prefer `python3.pkgs` over `python3Packages`?

Reply:
If overlays are done incorrect, those could point to different python versions, so we generally try to use python*.pkgs instead of python*Packages when it is already available.

Reply:
I see, thanks.